js/tabs/js/lumx.tabs_directive.js

```diff
@@ -107,11 +270,11 @@ angular.module('lumx.tabs', [])
                 direction = 'left';
             }
 
-            var tabsWidth = links.outerWidth(),
+            var tabsVisibleWidth = links.parent('.tabs').outerWidth(),
                 activeTab = links.find('.tabs-link').eq($scope.activeTab),
```

Comment:
I had a problem at the initialization of the component when I replaced the ```links.parent('.tabs')``` by the variable used in the controller. I'll take a closer look to see if I can avoid using ```link.parent('.tabs')```.